src/Chainweb/Pact/PactService.hs

```diff
@@ -885,11 +884,13 @@ execLocal cwtx preflight sigVerify rdepth = pactLabel "execLocal" $ do
                                             Right cr -> do
                                                 let cr' = hashPact5TxLogs cr
                                                 -- FIXME: Pact5, no warnings yet
```

Comment:
is this fixme still relevant?

Reply:
Probably better as a TODO than a FIXME. As far as I know, Pact 5 just doesn't have warnings right now.